circuit_knitting/cutting/cutting_decomposition.py

```diff
@@ -260,3 +267,105 @@ def decompose_observables(
     }
 
     return subobservables_by_subsystem
+
+
+def find_cuts(
+    circuit: QuantumCircuit,
+    optimization: dict[str, str | int],
+    constraints: dict[str, int],
+) -> tuple[QuantumCircuit, dict[str, Any]]:
+    """
+    Find cut locations in a circuit, given optimization settings and QPU constraints.
+
+    Args:
+        circuit: The circuit to cut
+        optimization: Settings for controlling optimizer behavior. Currently,
```

Comment:
I'd like to explicitly state the optimization and constraint options the user has here before merging